### PR TITLE
Contributors quiz callout banner

### PR DESCRIPTION
**Pull request by pettinarip**
## Description

Adds a banner for the contributor quiz to the `/contributing' page.

https://deploy-preview-12908--ethereumorg.netlify.app/en/contributing/

### CODE REVIEW

File: src/components/Banners/ContributorsQuizBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import { Box, Flex, FlexProps, Heading, Text } from "@chakra-ui/react"
+
+import { Image } from "@/components/Image"
+
+import { ButtonLink } from "../Buttons"
+
+import PplLearning from "@/public/ppl-learning.png"
+
+// TODO: refactor to use CalloutBanner component
```
Comment (pettinarip) [inline src/components/Banners/ContributorsQuizBanner.tsx:9]: https://github.com/ethereum/ethereum-org-website/pull/12911

**Review by corwintines — APPROVED** [review]
LGTM from code side

File: public/ppl-learning.png
Comment (minimalsm) [inline public/ppl-learning.png:1]: we should name this file properly `people-learning.png`. 

it's also quite large, does that matter?

Comment (pettinarip) [reply public/ppl-learning.png:1]: agree on naming

re: size, we should have the high res version of the image as the source. Then Next is going to resize it as needed.

**Review by wackerow — APPROVED** [review]
Looks great!

image

Thanks @pettinarip!

**Pull request merged**